browser/pages/selection/selection.html

```diff
@@ -15,14 +15,14 @@
           <div class="col-sm-4 channels-nav">
               <ul class="nav nav-tabs nav-stacked nav-pills" role="tablist">
                   <li id="{{tab.id}}-tab" ng-class="{'active': selectCtrl.channel_tab === tab.id}" ng-repeat="tab in selectCtrl.channelList">
-                      <a class="btn-lg" ng-click="selectCtrl.channel_tab = tab.id" href="">{{tab.name}}</a>
+                      <a class="btn-lg" ng-init="selectCtrl.componentsInChannel('containerDev')" ng-click="selectCtrl.componentsInChannel(tab.id)" href="">{{tab.name}}</a>
```

Comment:
We already have defined `channel_tab` initial value as 'containerDev'. There should not be any need to define again in ng-init.

Reply:
Thanks mohit I will make the changes :)

Reply:
If we remove 'containerDev' from ng-init then we can't select the other channels.

Reply:
ng-init is not the right way to use the scope value. It is not recommended a best practice. So we should not be using it. https://docs.angularjs.org/api/ng/directive/ngInit

Reply:
when we are using bind it's calling all component twice but when we are using `ngInit`  it calls all component only once, So I think we can use `ngInit` here.